frontend/packages/jobs/src/prompts/generateReview/generateReview.ts

```diff
@@ -41,10 +41,22 @@ Your JSON-formatted response must contain:
     - A score of 7-9 indicates minor issues or concerns
     - A score of 10 indicates perfect design with no issues
   - "reason": An explanation justifying the score provided, including what deductions were made and why. If no issues were found in a category, explicitly state that a score of 10 was given because no issues were identified.
-- A brief summary of the review in the "summary" field.
+- A concise summary in the "summary" field that:
+  - Describes the migration changes in about 1 line
+  - Highlights the most important issues or risks (if any exist); if no significant issues are found, highlight positive aspects of the migration instead
+  - Is about 3 lines in total length
 - A detailed and constructive overall review in the "bodyMarkdown" field.
   - The bodyMarkdown should be a markdown formatted string.
 
+Evaluation Criteria Details:
+- **Migration Safety:** Evaluates whether mechanisms are in place to ensure that all changes are atomically rolled back and system integrity is maintained even if migration operations (such as DDL operations and data migration) are interrupted or fail midway. This is a general safety indicator.
+- **Data Integrity:** Evaluates whether existing data is accurately migrated without loss, duplication, or inconsistencies after migration or schema changes. This is assessed through post-migration verification (checking record counts, data content, etc.) as a general data quality indicator.
+- **Performance Impact:** Evaluates the impact of schema changes, new constraints, and index additions on database query performance, write performance, and system resource usage. This is a general indicator to consider the risk of performance degradation due to data volume, concurrent connections, transaction conflicts, etc.
+- **Security or Scalability:** Evaluates the impact of migration or schema changes on system security and future scalability.
+  - **Security:** Includes risks such as storing sensitive information (passwords, etc.) in plain text or deficiencies in access control.
+  - **Scalability:** Evaluates the potential for performance degradation due to large-scale data processing, query delays, transaction conflicts, database locks, etc., as the system expands. This is a general perspective for evaluation.
```

Comment:
just note: https://github.com/liam-hq/liam/pull/1102#issuecomment-2775297299 PR-agent mentions other items. : `SQL injection vulnerabilities, inappropriate permission changes, or potential data exposure during migration.`

But for now,  I think current sentence is ok